client/my-sites/stats/stats-banners/index.jsx

```diff
@@ -143,7 +121,6 @@ export default connect( ( state, ownProps ) => {
 			state,
 			ownProps.siteId
 		),
-		isGSuiteStatsNudgeVisible: false,
```

Comment:
I'm not able to comment in GitHub on lines of code above but as far I can see we no longer need to retrieve the list of email forwards via `<QueryEmailForwards>` (this was introduced in https://github.com/Automattic/wp-calypso/pull/36286 to hide the banner when the user has active email forwards). This means we could also remove the `gsuiteDomainName` and `isAllowedToManageSite` props.